src/mono/wasm/runtime/binding_support.js

```diff
@@ -841,6 +914,67 @@ var BindingSupportLib = {
 			return result;
 		},
 
+		_resolve_class_ptr: function (ptr) {
+			if (typeof (ptr) !== "number")
+				throw new Error("Expected class pointer");
+
+			ptr = ptr | 0;
+			if (!ptr)
+				throw new Error("Expected class pointer");
+			return ptr;
+		},
+
+		extract_js_obj_with_converter: function (mono_obj, klass) {
```

Comment:
This is meant as a safer version of ```extract_js_obj_with_possible_converter``` but right now neither one is used anymore because refactorings made them unnecessary. I'd like to keep them for now and we can remove them later if we don't think they will ever be needed. The caller isn't really in a position to use just one or the other, since whether a given object has a converter is hard for a caller to decide. We could make it a single method with a 'throw on missing converter' parameter perhaps.

Reply:
I don't think we should introduce methods that are never used

Reply:
it not being used was a mistake, it's being used now